MMTabBarView Demo/MMTabBarView Demo/en.lproj/DemoWindow.xib

```diff
@@ -531,7 +532,7 @@
                                     <color key="backgroundColor" name="controlColor" catalog="System" colorSpace="catalog"/>
                                 </textFieldCell>
                             </textField>
-                            <popUpButton verticalHuggingPriority="750" id="9">
+                            <popUpButton verticalHuggingPriority="750" misplaced="YES" id="9">
```

Comment:
These misplaced attributes are like cancer in the xib-files. We should not include them anywhere if avoidable

Reply:
Same remark. Changed now.